src/Curator/ApplyPvpRules.hs

```diff
@@ -32,3 +32,7 @@ applyPvpRules constraints0 major minor = do
   let goPC name pc = pc { pcSource = goSource name $ pcSource pc }
   let packages = Map.mapWithKey goPC $ consPackages constraints0
   pure constraints0 { consPackages = packages }
+
+-- HELP
```

Comment:
Not sure what the "new" way of doing this is, could someone please assist?

Reply:
I'll have a look.

Reply:
I'm pretty confused, why is this breaking now? It looks like `ltsSnapshotLocation` code is already being used on `master`.

Reply:
OK, I guess there's a `pantry` update implicitly happening here too. I think this is the correct new version:

```haskell
ltsSnapshotLocation :: Int -> Int -> RawSnapshotLocation
ltsSnapshotLocation x y = defaultSnapshotLocation $ LTS x y
```

Does that solve it for you? It seems to work on my system.

Reply:
yeah we were using a commit between 0.4.0.1 and 0.5.0.0 it seems.

fixed, thanks!